content/english/blog/road_to_ubiquity.md

```diff
@@ -0,0 +1,75 @@
+
+However, since everyone making an application or platform decision at this level will be opting into the vendor's SDK, it is necessary for us to make this SDK as optimized for developers as as possible, making the following first-party activities and capabilities "brain-dead simple" _by default_:
+
+* HTTP Server
+* HTTP Client
+* Message Broker
+* Blob Store (e.g. S3)
+* Key Value
+* SQL DB
+* Actor-to-Actor Calls
+
+This wasmCloud developer SDK will be a thin wrapper on top of the generated interface code designed to smooth and optimize the developer experience and ease of use. Using this SDK will insulate developers from the churn and change happening at the engine and specification level. As `wit` and the component model mature, we'll simply integrate that work into our SDK, sparing developers from as many breaking changes as possible.
+
+### The Component Model
+Once WebAssembly is boring and ubiquitous and tooling exists for all manner of scenarios and the component model has become mature, developers will be able to rapidly adapt their existing code bases to any number of platform providers because consuming them becomes just a matter of generating client code against wit models.
+
+It's worth noting that practically speaking, most developers won't be doing this. The "easy default" case will likely be that developers will choose to use the ready-made, shrink-wrapped SDK of whichever platform they choose to run their code on.
+
+However, being built on the component model gives us the ability to rapidly adapt to any changes in the community and, more importantly, assert that there is _no vendor lock-in for consuming wasmCloud and Cosmonic services_, only the choice of relying on any of the portable component modules that represent the developer-facing contract for capability providers.
+
+There is an effort within the standards community to describe a means by which a wasm component can specify the requirements of its host in terms of the components it uses. At the moment, these specifications are called _world files_, but that nomenclature could change at any moment.
+
+The concept of this type of requirements specification is subtle yet _incredibly_ powerful. In a hypothetical future world where WebAssembly and the component model are both boring, a module running in "vendor A"'s cloud that requires a key-value store could be moved to "vendor B"'s cloud without design change or recompilation, provided both vendors support exposing the WASI key-value component in their "world". This could truly be a fulfillment of the portability promise beyond just escaping from the confines of OS and CPU coupling.
+
+In this bright idyllic near-future world, people write code _one way_ and then the selection of where they deploy their code becomes one of personal preference, financial concerns, and available vendor features. No longer will we be locked into a particular vendor because that's the SDK we started experimenting with a year ago.
```

Comment:
Contrary to my previous opinions, this section feels like it needs a paragraph to "bring it back down to earth." This paragraph is a pretty bold claim (that I believe is distinctly possible) but it might be good to talk about what is required and how it might be a little bumpy to get there